src/client.cpp

```diff
@@ -120,8 +120,8 @@ ADDON_STATUS ADDON_GetStatus()
   CLockObject lock(g_mutex);
 
   // Check that we're still connected
-  if (m_CurStatus == ADDON_STATUS_OK && !tvh->IsConnected())
-    m_CurStatus = ADDON_STATUS_LOST_CONNECTION;
+  if (m_CurStatus == ADDON_STATUS_OK && !tvh->IsSuspended() && !tvh->IsConnected())
+    return ADDON_STATUS_LOST_CONNECTION;
```

Comment:
@FernetMenta returning ADDON_STATUS_LOST_CONNECTION triggers the "Connection lost" notification in Kodi, which can be disabled by checking the "Do not show connection warnings" setting in Kodi. I don't see why we'd have to remove the status, many clients have a concept of connecting to a backend which implies there is a possibility that the connection is lost. pvr.iptvsimple obviously doesn't have this concept, but it's not a true PVR addon either.